pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -428,15 +496,20 @@ public void setRetentionControllerFrequencyInSeconds(int retentionFrequencyInSec
   }
 
   /**
-   * Returns the config value for controller.offline.segment.interval.checker.frequencyInSeconds if it exists.
-   * If it doesn't exist, returns the segment level validation interval. This is done in order to retain the current behavior,
-   * wherein the offline validation tasks were done at segment level validation interval frequency
-   * The default value is the new DEFAULT_OFFLINE_SEGMENT_INTERVAL_CHECKER_FREQUENCY_IN_SECONDS
+   * Returns the config value for controller.offline.segment.interval.checker.frequencyInSeconds if
```

Comment:
We should check the new one first and then the old one, right?

Reply:
Missed this, will update the docs.